### ISSUE TITLE
Print alignment

### ISSUE THREAD

**Issue by Permafacture** [libnano/primer3-py#28]
Is there any interest in adding the ascii alignment representation to primer3-py?  I can access it from the compiled C code like:

```
./ntthal -s1 AATGATACGGCGACCACCGATACCTCTGCAAACTTGGAGATGACC -s2 AAATTTTT
Calculated thermodynamical parameters for dimer:	dS = -43.5049	dH = -13700	dG = -206.968	t = -101.184
SEQ	AATGATACGGCGACCACCGATACCTCTGC   CTTGGAGATGACC
SEQ	                             AAA
STR	                             TTT
STR	                           TT   AAA---------- 
```
I would benefit from this functionality and I'm wondering if any thought has already gone into how this would be implemented.

**Comment by Permafacture** [comment]
This is coming up for me again. Any thoughts or reactions?

**Comment by benpruitt** [comment]
I _think_ we might have modified the library in a way that would make this more challenging than it would appear, but let me take some time to look into it this week and I'll let you know.

**Comment by Permafacture** [comment]
Thanks! Honestly, just the offset between the two sequences would be fine, and the visualization would be easy to reconstruct in python

**Comment by benpruitt** [comment]
Resolved in #31
